### PR TITLE
Wait for user permissions in test-cmd.sh

### LINKED ISSUE #6419: user cannot create template in project flake
```
FAILURE after 0.050s: hack/../test/cmd/templates.sh:65: executing 'oc create -f examples/sample-app/application-template-dockerbuild.json' expecting success: the command returned the wrong error code
There was no output from the command.
Standard error from the command:
Error from server: error when creating "examples/sample-app/application-template-dockerbuild.json": User "test-user" cannot create templates in project "test-template-project"
[FAIL] !!!!! Test Failed !!!!
```
https://ci.openshift.redhat.com/jenkins/job/merge_pull_requests_origin/4416/consoleFull

### PR DESCRIPTION
**Pull request by liggitt**
Fixes #6419

### CODE REVIEW

**Comment by liggitt** [comment]
@kargakis PTAL

**Comment by liggitt** [comment]
[test]

File: test/cmd/templates.sh [hunk 1]
```diff
@@ -62,6 +62,7 @@ os::cmd::expect_success 'oc create -f examples/sample-app/application-template-d
 os::cmd::expect_success 'oc policy add-role-to-user admin test-user'
 os::cmd::expect_success 'oc login -u test-user -p password'
 os::cmd::expect_success 'oc new-project test-template-project'
+os::cmd::try_until_success 'oc policy who-can create imagestreams' 'test-user'
```
Comment (kargakis) [inline test/cmd/templates.sh]: The error is about creating a template and not an image stream.

Comment (liggitt) [inline test/cmd/templates.sh]: doesn't matter, I'm just waiting until one of the permissions associated with being a project admin is reported

**Comment by kargakis** [comment]
lgtm [merge]

File: test/cmd/templates.sh (hunk 1 above)
Comment (liggitt) [inline test/cmd/templates.sh]: switched to a simpler check and added a comment

Comment (kargakis) [inline test/cmd/templates.sh]: > doesn't matter, I'm just waiting until one of the permissions associated with being a project admin is reported

I see...

**Comment by kargakis** [comment]
new-app flake
```
FAILURE after 0.377s: hack/../test/cmd/newapp.sh:51: executing 'oc new-app --search rub' expecting success and text 'Tags:\s+2.0, 2.2, latest': the output content test failed
Standard output from the command:
Templates (oc new-app --template=<template>)
-----
ruby-helloworld-sample
  Project: cmd-newapp
  This example shows how to create a simple ruby application in openshift origin v3

Image streams (oc new-app --image-stream=<image-stream> [--code=<source>])
-----
ruby
  Project: cmd-newapp
  Tags:    <none>

There was no error output from the command.
[FAIL] !!!!! Test Failed !!!!
```
[merge]

**Comment by kargakis** [comment]
[merge]

**Pull request merged**